### DISCUSSION TITLE
Can a repositorybe visible only for SSO login?

### DISCUSSION THREAD

**Discussion by LiorBanai** [community/community#21812] [Enterprise]
<p>Can an organisation (that uses SSO) set a repository to be viewable only after a user sign in to the organisation using its SSO? what about outside collaborators?</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/liorbanai">@liorbanai</a>,</p>
<p>Thanks for being here, on GHES <a href="https://github.com/enterprise" rel="nofollow noopener">GitHub Enterprise is provided via two solutions</a>, one is hosted on our infrastructure via <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> and the other is a self-hosted server platform.</p>
<p>For the <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> Enterprise Cloud product, each staff member needs to have their own individual <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> login. Their personal <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a> login is then linked to an identity in your IdP the first time they access the Organization on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>. The individual account itself however remains with the user even after they leave your organization, although their access to your private repositories is then disabled.</p>
<p>As the GitHub login belongs to the individual, there is no way to prevent them from accessing other resources hosted on <a href="http://GitHub.com" rel="nofollow noopener">GitHub.com</a>.</p>
<p>Alternatively identity integration via SAML, LDAP, or CAS is also available in our self-hosted GitHub Enterprise Server product. GitHub Enterprise Server is a standalone platform which only hosts your own users and repositories. It must be deployed either to your <a href="https://help.github.com/enterprise/admin/guides/installation/setting-up-a-github-enterprise-server-instance/" rel="nofollow noopener">own virtualisation infrastructure, or a supported cloud hosting provider</a>.</p>

**Comment by LiorBanai** [comment]
<p>This is helpful. Thanks!</p>
